wire-library/wire-grpc-client/src/commonMain/kotlin/com/squareup/wire/internal/GrpcEncoder.kt

```diff
@@ -29,7 +27,7 @@ internal sealed class GrpcEncoder(val name: String) {
   }
 
   internal object GzipGrpcEncoder : GrpcEncoder("gzip") {
-    override fun encode(sink: BufferedSink) = sink.gzip().buffer()
+    override fun encode(sink: BufferedSink) = sink.asGzip().buffer()
```

Comment:
@oldergod any idea why this one is buffered but the decoder is not?

Reply:
I don't recall. Maybe @swankjesse does.

Reply:
Typically it’s ’cause that’s what the callers want.